packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -360,6 +439,12 @@ class DDSMastheadComposite extends LitElement {
   @property({ attribute: false })
   navLinks?: MastheadLink[];
 
+  /**
+   * Data for l1.
+   */
+  @property({ attribute: false })
+  l1Data? = { title: '', url: '', menuItems: [] };
```

Comment:
Given we don't want to render L1 when `this.l1Data` is `undefined`, I think we don't want to define the default value:

```suggestion
  l1Data?: MastheadL1;
```